common/lib/xmodule/xmodule/modulestore/tests/test_semantics.py

```diff
@@ -361,6 +418,32 @@ class TestSplitDirectOnlyCategorySemantics(DirectOnlyCategorySemantics):
     MODULESTORE = SPLIT_MODULESTORE_SETUP
     __test__ = True
 
+    @ddt.data(*TESTABLE_BLOCK_TYPES)
+    @XBlockAside.register_temp_plugin(AsideTest, 'test_aside')
+    @patch('xmodule.modulestore.split_mongo.caching_descriptor_system.CachingDescriptorSystem.applicable_aside_types',
+           lambda self, block: ['test_aside'])
+    def test_create_with_asides(self, block_type):
+        self._do_create(block_type, with_asides=True)
+
+    @ddt.data(*TESTABLE_BLOCK_TYPES)
+    @XBlockAside.register_temp_plugin(AsideTest, 'test_aside')
+    @patch('xmodule.modulestore.split_mongo.caching_descriptor_system.CachingDescriptorSystem.applicable_aside_types',
+           lambda self, block: ['test_aside'])
+    def test_update_asides(self, block_type):
+        block_usage_key = self._do_create(block_type, with_asides=True)
+        test_data = self.DATA_FIELDS[block_type]
+
+        with self.store.branch_setting(ModuleStoreEnum.Branch.draft_preferred):
+            block = self.store.get_item(block_usage_key)
+            aside = self._get_aside(block)
```

Comment:
The intention of these tests is to work only using the public `Modulestore` interface. So, in this case, it would be better to assert that `block.get_asides()` returns the expected aside, rather than using the `_get_aside` method which isn't public.